### PR TITLE
Support SVG images

### PR DESCRIPTION
**Pull request by tmikoss**
Given a SVG placeholder, current image insertion code could handle SVGs pretty easily - it's just the difference between modifying `a:blip` vs `asvg:svgBlip` tags as far as I can tell.

### CODE REVIEW

File: test/sablon_test.rb [hunk 1]
```diff
@@ -187,3 +187,25 @@ def test_generate_document_from_template
     assert_equal "Filename: 'clone' has no discernable extension", e.message
   end
 end
+
+class SablonImagesTest < Sablon::TestCase
```
Comment (stadelmanma) [inline test/sablon_test.rb:191]: Can you change this to `SablonSvgImageTest`? That way the name doesn't conflict with the above class.

**Review by stadelmanma — CHANGES_REQUESTED** [review]

**Comment by stadelmanma** [comment]
@tmikoss this looks good just one minor change needed in the test file. Thanks for adding support for SVGs! I don't think I knew that they used a different node.

File: test/sablon_test.rb (hunk 1 above)
Comment (tmikoss) [reply test/sablon_test.rb]: Copy-paste strikes again. Fixed.

**Comment by tmikoss** [comment]
Couple things to consider:
* is `image.name` always a filename? Appears to me it is, since content type is being inferred from it;
* is supported version of Word left up to the end user, based on the features they use? As far as I can tell, it is supported starting with Word 2016;
* this is a "works on my machine" kinda solution - I'm no expert on the docx format. Then again, that's the tag Word generates when you drag a SVG into a document, we're just updating the reference / parameters attributes, so it should work fine.

**Comment by tmikoss** [comment]
@stadelmanma anytning else I need to update?

**Review by stadelmanma — APPROVED** [review]

**Comment by stadelmanma** [comment]
@tmikoss sorry about the delay, life takes over occasionally. The PR looks great thanks for adding this! And you are correct, the MS Word version is left up to the user but so far we haven't had any issues regarding that "gotcha". (Not sure if we could do anything about it anyways)

**Pull request merged**